src/api/LoginApi.js

```diff
@@ -0,0 +1,23 @@
+import { Axios } from './Axios';
+
+export const login = async (data) => {
+  try {
+    const response = await Axios.post('/auth/login', {
+      id: data.id,
+      pw: data.pw,
+    });
+    console.log(response);
+    return response;
```

Comment:
요러한 API 함수들에선 아래처럼 변경하면 사용하는 쪽에서 좀 편하고, 더 와닿지 않을까 생각 드는데 어떻게 생각하시나용

```suggestion
    return response.data
```

Reply:
대신 받아오는 걸 data.status 로 받아왔었는데 같이 수정하겠습니다 !!